packages/Input/src/ToggleInputType/ToggleInputType.tsx

```diff
@@ -1,57 +1,60 @@
-import React, { useState } from 'react';
+import React, { useState } from "react";
 
 interface IToggleInputTypeProps {
   type: string | undefined;
-  toggleType: (newType: string) => void
+  toggleType: (newType: string) => void;
 }
 interface IPasswordState {
-  icon: string,
-  status: string,
-  text: string
+  icon: string;
+  status: string;
+  text: string;
 }
 
 const ToggleInputType = ({
   type,
   toggleType
 }: IToggleInputTypeProps): JSX.Element => {
-
   const passwordDefaultClasses: IPasswordState = {
-    icon: 'visibility_off',
-    status: 'active',
-    text: 'Show'
+    icon: "visibility_off",
+    status: "active",
+    text: "Show"
   };
 
   const passwordActiveClasses: IPasswordState = {
-    icon: 'visibility_off',
-    status: 'active',
-    text: 'Show'
+    icon: "visibility",
+    status: "hide",
+    text: "Hide"
   };
 
-  const [passwordClasses, setPasswordState] = useState<IPasswordState>(passwordDefaultClasses);
+  const [passwordClasses, setPasswordState] = useState<IPasswordState>(
+    passwordDefaultClasses
+  );
 
-  const isPassword = type === 'password';
+  const isPassword = type === "password";
 
   const handleToggleClasses = (): void => {
     if (isPassword) {
       setPasswordState(passwordActiveClasses);
-      toggleType('text');
+      toggleType("text");
```

Comment:
Should be a constant

Reply:
Agree, all hardcoded strings in general should be stored in a constant.